File: app/src/main/java/com/pnuema/simplebible/statics/HtmlUtils.java
package com.pnuema.simplebible.statics;

import android.os.Build;
import android.text.Html;
import android.text.Spanned;

public final class HtmlUtils {
    private HtmlUtils() {
    }

    @SuppressWarnings("deprecation")
    public static Spanned fromHtml(String source) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            return Html.fromHtml(source, Html.FROM_HTML_MODE_LEGACY);
        } else {
            return Html.fromHtml(source);
        }
    }
}

File: app/src/main/java/com/pnuema/android/bible/statics/HtmlUtils.kt
package com.pnuema.android.bible.statics

import android.os.Build
import android.text.Html
import android.text.Spanned

object HtmlUtils {

    fun fromHtml(source: String): Spanned {
        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            Html.fromHtml(source, Html.FROM_HTML_MODE_LEGACY)
        } else {
            Html.fromHtml(source)
        }
    }
}
